### PR TITLE
[IMP] product_brand: Move menu

### PR DESCRIPTION
**Pull request by chienandalu**
- Product brand menu is unavailable if _Products with variants_ option is unset. This patch moves it to _Configuration > Products_, where other similar menus (like product categories) fall.
- Some styling tweaks on README and others.
- Remove of .pot file

cc @Tecnativa

### CODE REVIEW

File: product_brand/__openerp__.py [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
-              ', MONK Software, Odoo Community Association (OCA)'
-              ', SerpentCS Pvt. Ltd.',
+    'summary': 'Product Brand Manager',
+    'author': 'NetAndCo, '
+              'Akretion, '
+              'Prisnet Telecommunications SA, '
+              'MONK Software, '
+              'SerpentCS Pvt. Ltd., '
+              'Tecnativa, '
```
Comment (pedrobaeza) [inline product_brand/__openerp__.py]: Don't add here Tecnativa as there are too much characters (maximum is 128)

**Comment by chienandalu** [comment]
@pedrobaeza fixed

**Review by pedrobaeza — APPROVED** [review]
@luismontalba @cubells please review

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

**Comment by cubells** [comment]
@chienandalu 

can you change 

```<record model="ir.actions.act_window" id="action_product_brand">
 <field name="name">Brand</field>
 <field name="res_model">product.brand</field>
 <field name="view_type">form</field>
 <field name="view_mode">kanban,form,tree</field>
 </record>
```
by

```<record model="ir.actions.act_window" id="action_product_brand">
            <field name="name">Brand</field>
            <field name="res_model">product.brand</field>
            <field name="view_type">form</field>
            <field name="view_mode">kanban,form,tree</field>
            <field name="help" type="html">
                <p class="oe_view_nocontent_create">
                    Create a Product Brand.
                </p>
            </field>
        </record>
```
To avoid this form:

seleccio_314

It is awful.

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by chienandalu** [comment]
@pedrobaeza @cubells Changes done and green CIs

**Comment by pedrobaeza** [comment]
Please squash and I'll merge.

**Comment by chienandalu** [comment]
@pedrobaeza Done

**Pull request merged**